dvc/repo/__init__.py

```diff
@@ -129,10 +132,10 @@ def _ignore(self):
         updater = Updater(self.dvc_dir)
 
         flist = [
-            self.lock.lock_file,
+            self.lock.lockfile,
             self.config.config_local_file,
             updater.updater_file,
-            updater.lock.lock_file,
+            updater.lock.lockfile,
```

Comment:
Looks like you've forgot to add `.dvc/tmp` here, so it is gitingored

Reply:
Added